feat(auth): expose access token expiry under auth settings (#48236)
supabase/supabase: apps/docs/content/guides/auth/sessions.mdx

### What are recommended values for access token (JWT) expiration?

> Most applications should use the default expiration time of 1 hour. This can be customized under the **Access Tokens** section in the [Auth settings](/dashboard/project/_/auth/sessions).

Reviewer: I suggest either doing **Access Token > Auth settings** or rephrasing to:

```suggestion
Most applications have an expiration time of 1 hour by default.

You can customize this value in the [Auth settings](/dashboard/project/_/auth/sessions).
```
